src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -37,6 +53,88 @@ public abstract partial class ECDsa : System.Security.Cryptography.AsymmetricAlg
         public bool VerifyData(System.IO.Stream data, byte[] signature, System.Security.Cryptography.HashAlgorithmName hashAlgorithm) { return default(bool); }
         public abstract bool VerifyHash(byte[] hash, byte[] signature);
     }
+    public abstract partial class ECExplicitCurve : System.Security.Cryptography.ECCurve
+    {
+        public byte[] A;
+        public byte[] B;
+        public System.Security.Cryptography.ECCurveType CurveType;
+        public System.Security.Cryptography.ECPoint G;
+        public byte[] h;
+        public System.Security.Cryptography.HashAlgorithmName Hash;
+        public byte[] n;
```

Comment:
We should use the friendly names of n, h, and S: Order, Cofactor, and Seed, respectively.